### PR TITLE
[SQL] Correct bugbear leathercraft desynth

### PR DESCRIPTION
**Pull request by jamesbradleym**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Changes the quantities recieved for the desynth tiers from 2 to 3 for NQ -> HQ2 per era resources:
http://ffxi.somepage.com/itemdb/5793
http://www.taultunleashed.com/final-fantasy-xi-discussions/desynthing-for-fun-and-profit-t9152.html

Closes: https://github.com/HorizonFFXI/HorizonXI-Issues/issues/1834

## Steps to test these changes

Update sql file (synth_recipes.sql) using dbtool

!additem 1624
!additem 4098

Desynth bugbear mask

### CODE REVIEW

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -2816,7 +2816,7 @@ INSERT INTO `synth_recipes` VALUES (41513,0,2016,0,0,0,0,35,0,0,0,4103,4245,635,
 INSERT INTO `synth_recipes` VALUES (41514,0,2018,0,0,0,0,35,0,0,0,4103,4245,635,635,635,859,859,859,2129,4509,851,851,851,851,3,3,3,3,'Ram Leather');
 INSERT INTO `synth_recipes` VALUES (41515,0,0,0,0,0,0,35,0,0,0,4103,4245,635,859,4509,0,0,0,0,0,851,851,851,851,1,1,1,1,'Ram Leather');
 INSERT INTO `synth_recipes` VALUES (41516,0,0,0,0,0,0,35,0,0,0,4099,4241,850,12882,0,0,0,0,0,0,14265,14266,14266,14266,1,1,1,1,'San d\'Orian Trousers');
-INSERT INTO `synth_recipes` VALUES (41517,1,0,0,0,0,0,35,0,0,0,4098,4240,1624,0,0,0,0,0,0,0,851,823,823,823,3,2,2,2,'Bugbear Mask (desynth)');
+INSERT INTO `synth_recipes` VALUES (41517,1,0,0,0,0,0,35,0,0,0,4098,4240,1624,0,0,0,0,0,0,0,851,823,823,823,3,3,3,2,'Bugbear Mask (desynth)');
```
Comment (Xaver-DaRed) [inline sql/synth_recipes.sql]: This is correct regarding the quantity changes, but is incorrect regarding the items rewarded.

Item ID 851 (Ram Leather) should be rewarded when NQ, HQ1 and HQ2
Item ID 823 (Gold Thread) should be rewarded when HQ3

So the change should be this...

```sql
INSERT INTO `synth_recipes` VALUES (41517,1,0,0,0,0,0,35,0,0,0,4098,4240,1624,0,0,0,0,0,0,0,851,851,851,823,3,3,3,2,'Bugbear Mask (desynth)');
```

**Review by Xaver-DaRed — CHANGES_REQUESTED** [review]
Please change the 2 wrong item IDs

File: sql/synth_recipes.sql (hunk 1 above)
Comment (jamesbradleym) [reply sql/synth_recipes.sql]: Thanks for catching that! Fixed.

**Review by Xaver-DaRed — APPROVED** [review]

**Comment by jamesbradleym** [comment]
@Xaver-DaRed First time doing this, let me know if that squash looks correct?

**Comment by Xaver-DaRed** [comment]
Yeah, perfect, ty

**Pull request merged**